src/cpknextgen/transformation.py

```diff
@@ -187,29 +190,29 @@ def __init__(self, data: np.ndarray,
         if len(np.unique(self.boundaryless_data)) < 3 and jitter_kernel == 'no_jitter':
             raise InvalidDataError("More unique values are required to calculate capability indexes.")
 
-        self.lower_specification, self.upper_specification, self.standard_computation = self._check_limits(
+        self.lower_normalization_limit, self.upper_normalization_limit, self.standard_computation = self._check_limits(
             self.boundaryless_data,
             lower_specification_limit,
             upper_specification_limit,
             limit_quantiles,
             lower_boundary,
             upper_boundary)
 
-        self.lower_specification_transformed = None
-        self.upper_specification_transformed = None
+        self.lower_normalization_limit_transformed = None
+        self.upper_normalization_limit_transformed = None
 
         self.lower_boundary_transformed = None
         self.upper_boundary_transformed = None
 
         self.specification_to_boundaries = self.relative_to_boundaries(
```

Comment:
Shouldn't this be also renamed to `self.normalization_to_boundaries`?

Reply:
No, "normalization_to_boundaries" could be new name for "specification_to_boundaries". "lower_normalization_limit_ transformed" could be renamed to "lower_normalization_to_yeo" but this would need renaming of other variables as well.